go-controller/pkg/ovn/service.go

```diff
@@ -43,38 +43,33 @@ func (ovn *Controller) syncServices(services []interface{}) {
 			continue
 		}
 
-		if !util.IsClusterIPSet(service) {
```

Comment:
why is this being removed?

Reply:
Must have been a mind-slip. I will rectify